_includes/tutorial-beginner-menu.html

```diff
@@ -5,10 +5,10 @@
         <ul>
             <li><a href="{{docu}}/index.html">Overview</a></li>
             <li><a href="{{docu}}/uis.html">User Interfaces</a></li>
-            <li><a href="{{docu}}/installing.html">Installing an extension</a></li>
-            <li><a href="{{docu}}/persistence.html">Working with persistence</a></li>
-            <li><a href="{{docu}}/rulesscripts.html">Working with rules and script</a></li>
-            <li><a href="{{docu}}/logs.html">Looking to the logs</a></li>
+            <li><a href="{{docu}}/installing.html">Installing an Extension</a></li>
+            <li><a href="{{docu}}/configuring.html">Configuring Functionality</a></li>
+            <li><a href="{{docu}}/definingui.html">Defining User Interface</a></li>
+            <li><a href="{{docu}}/logs.html">Looking to the Logs</a></li>
```

Comment:
Do you think the document names are a good match? I feel like nouns are more common for html names!? Could be wrong :)
"installextension.html" or "uisetup.html"?